src/test/java/com/linkedin/kmf/services/ConsumeServiceTest.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.services;
+
+import com.linkedin.kmf.services.configs.CommonServiceConfig;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.atomic.AtomicReference;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.testng.annotations.Test;
+
+
+/**
+ * Unit Testing for the Consume Service Class.
+ */
+public class ConsumeServiceTest {
+  private static final String BROKER_LIST = "localhost:9092";
+  private static final String ZK_CONNECT = "localhost:2181";
+  private static final String TOPIC = "kafka-monitor-topic-test";
+
+  @Test
+  public void lifecycleTest()
+      throws ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException,
```

Comment:
Giant exception lists are hard to read and not very useful.  Also the test framework treats all exceptions the same unless it is told not to.  So it's safe to make this just throw Exception.